stdlib/benchmarks/builtin/bench_sort_scalars.mojo

```diff
@@ -0,0 +1,238 @@
+# ===----------------------------------------------------------------------=== #
+# Copyright (c) 2024, Modular Inc. All rights reserved.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions:
+# https://llvm.org/LICENSE.txt
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ===----------------------------------------------------------------------=== #
+
+# RUN: %mojo %s
```

Comment:
```suggestion
# RUN: %mojo %s -t
```

Do you mind adding the `-t`?  This option gets parsed by the benchmarking package internally to run it in "test mode" to speed up CI.  This will be clearer once we can open source the `benchmark` module in the future.